scalafmt-tests/src/test/resources/newlines/beforeConfigSingleArgParenLambdaParamsF_danglingF.stat

```diff
@@ -216,9 +193,9 @@ def f = {
 >>>
 EventFilter[ActorInitializationException](occurrences = 1) intercept {
   intercept[akka.actor.ActorInitializationException] {
-    wrap(
-      result ⇒
-        actorOf(Props(new OuterActor(
+    wrap(result ⇒
+      actorOf(
```

Comment:
This split is the only thing I couldn't fix, played with penalties in that match branch where I added the "preserve no-split in assign" logic.

Specifically, the sum `nestedPenalty + lhsPenalty` seemed to be constant for each of these nested calls which doesn't indicate a preference between them.

I tried `2 * nestedPenalty + lhsPenalty` to try and split earlier but it ultimately produced too many changes. I also tried `nestedPenalty + 2 * lhsPenalty` to prefer a later split, but it was also not working as well. So I gave up.